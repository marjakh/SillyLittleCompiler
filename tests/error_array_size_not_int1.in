let a = new Array(0);
let b = new Array(a);