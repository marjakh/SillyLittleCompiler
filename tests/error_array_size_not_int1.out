RuntimeError: Array size not an int
